lib/screens_web/views/v2/audio/reconstructed_alert_fullscreen_view.ex

```diff
@@ -0,0 +1,206 @@
+defmodule ScreensWeb.V2.Audio.ReconstructedAlertFullscreenView do
+  use ScreensWeb, :view
+
+  alias Screens.Alerts.Alert
+  alias Screens.Util
+
+  def render("_widget.ssml", alert) do
+    ~E|<p><%= render_banner(alert) %><%= render_alert(alert) %></p>|
+  end
+
+  def render_banner(%{
+        unaffected_routes: _unaffected_routes
+      }),
+      do: nil
+
+  def render_banner(%{
+        region: _region,
+        routes: routes
+      }) do
+    if routes |> hd() |> Map.has_key?(:headsign) do
+      destinations = Enum.map(routes, fn route -> route.headsign end)
+
+      if length(destinations) < 3 do
+        ~E|Attention, riders to <%= Util.format_name_list_to_string_audio(destinations) %>. |
+      else
+        ~E|Attention, riders. |
+      end
+    else
+      ~E|Attention, <%= hd(routes).route_id %> riders. |
+    end
+  end
+
+  def render_banner(_), do: nil
+
+  def render_alert(
+        %{
+          effect: :delay,
+          issue: issue
+        } = alert
+      ) do
+    ~E|<%= issue %><%= render_cause(alert.cause) %>.|
+  end
+
+  # Downstream closure
+  def render_alert(%{
+        region: :outside,
+        effect: :station_closure,
+        routes: route_svg_names,
+        cause: cause,
+        stations: stations
+      }) do
+    ~E|<%= get_line_name(route_svg_names) %> trains are skipping <%= Util.format_name_list_to_string_audio(stations) %><%= render_cause(cause) %>. Please seek an alternate route.|
+  end
+
+  # Downstream shuttle
+  def render_alert(%{
+        region: :outside,
+        effect: :shuttle,
+        routes: route_svg_names,
+        endpoints: endpoints,
+        cause: cause
+      }) do
+    ~E|Shuttle buses replace <%= get_line_name(route_svg_names) %> trains between <%= elem(endpoints, 0) %> and <%= elem(endpoints, 1) %><%= render_cause(cause) %>. All shuttle buses are accessible.|
+  end
+
+  # Downstream suspension
+  def render_alert(%{
+        region: :outside,
+        effect: :suspension,
+        routes: route_svg_names,
+        endpoints: endpoints,
+        cause: cause
+      }) do
+    ~E|There are no <%= get_line_name(route_svg_names) %> trains between <%= elem(endpoints, 0) %> and <%= elem(endpoints, 1) %><%= render_cause(cause) %>. Please seek an alternate route.|
+  end
+
+  # Boundary shuttle
+  def render_alert(%{
+        region: :boundary,
+        effect: :shuttle,
+        issue: issue,
+        routes: route_svg_names,
+        endpoints: endpoints,
```

Comment:
Could pattern-match the two values so that they have names and you don't need to use `elem/2`: `endpoints: {left_endpoint, right_endpoint},`

Here and below.

Reply:
Oh geez, yep